src/comp/comp.py

```diff
@@ -24,48 +24,49 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [humans[i].name for i in range(len(humans)) if humans[i].name[0] == 'D']
```

Comment:
like we discussed in 1:1, you can simplify this to
`a = [i.name for i in humans if i.name[0] =='D']`

Basically we want to iterate through the objects themselves instead of having to use their indices in the `humans` array.